### PR TITLE
Update cards.ini

### PR DESCRIPTION
**Pull request by szme**
Added a few more aliases for cards.ini

### CODE REVIEW

File: cards.ini [hunk 1]
```diff
@@ -49,3 +49,51 @@ j how = Jackson Howard
...
+doof = Diversion of Funds
+fugu = Puffer
+knock knock = No One Home
+just in time = Compile
+godzilla = Gbahali
+mopus = Magnum Opus
+saccon = Sacrificial Construct
+tb = Tollbooth
+petty = Punitive Counterstrike
+sft = Successful Field Test
```
Comment (Mezuzi) [inline cards.ini]: Going to ask to strike this out - this commonly also refers to Shipment from Tennin as well.

File: cards.ini [hunk 2]
```diff
@@ -49,3 +49,51 @@ j how = Jackson Howard
...
+hail = Hydra
+free willy = Darwin
+ttw = The Turning Wheel
+sauron = Hunter
+darude = Sand Storm
+hoot = Owl
+piot = Peace in Our Time
+nice car = Sports Hopper
+sweet car = Advanced Concept Hopper
+folding at home = Data Folding
```
Comment (Mezuzi) [inline cards.ini:61]: "rezeki at home"? perhaps? Otherwise this doesn't make much sense.

**Review by Mezuzi — CHANGES_REQUESTED** [review]
Thanks for the PR! Unfortunately there are quite a few aliases that seem more like memes and less like actual natural jargon used by Netrunner players. A few memes is good here and there but a few of these are more obscure or heavily reaching, and I don't think the one-word aliases are worth polluting real searches.

I'm going to have to request the following one and two word aliases be removed before I can merge this in (or a good enough persuasive argument for me to include them 😉):

- flowers for
- hail (gets confused for Hailstorm)
- free willy (you could make a good argument for this one to be re-alised to Secure And Protect) 
- overwatch
- netflix
- wow
- rebecca
- godzilla
- petty
- sft
- snp
- bong
- 911
- bollywood

File: cards.ini (hunk 2 above)
Comment (szme) [reply cards.ini:61]: Hi, this is actually a reference to: 
https://foldingathome.org/

It involves installing a software that uses spare processing power to help scientists unravel the mysteries of nature. Sounds kind of like Data Folding, no? :)

File: cards.ini (hunk 1 above)
Comment (szme) [reply cards.ini]: Heard the commentators refer to this to SFT during stream, but makes sense. Thanks!

**Comment by szme** [comment]
I have made the necessary removals. Thanks!

File: cards.ini (hunk 2 above)
Comment (Mezuzi) [reply cards.ini:61]: That's fair! Feel free to keep this one then. =D

**Review by Mezuzi — APPROVED** [review]
Thanks for the changes! Looks good. 👍

**Pull request merged**